### PR TITLE
Improve planet texts

### PR DESCRIPTION
**Pull request by ivalaginja**
This PR improves several of the already existing planet texts and provides a better, albeit not full, overview of the solar system and beyond.

This contains:
- a new file about Saturn
- changes in Earth and Mars files
- getting rid  of Venus, because who needs that hell of a place

### CODE REVIEW

File: Mars.txt [hunk 1]
```diff
@@ -1,9 +1,12 @@
 Der Mars ist, von der Sonne aus gezählt, der vierte Planet im Sonnensystem und der äußere Nachbar der Erde. Er zählt
 zu den erdähnlichen (terrestrischen) Planeten.
 
-Sein Durchmesser ist mit knapp 6800 Kilometern etwa halb so groß wie der der Erde, sein Volumen beträgt gut ein
+Sein Durchmesser ist mit knapp 6800 Kilometern etwa halb so groß wie der der Erde (Earth), sein Volumen beträgt gut ein
 Siebtel des Erdvolumens. Damit ist der Mars nach dem Merkur der zweitkleinste Planet des Sonnensystems, hat jedoch
-eine vielfältige Geologie und die höchsten Vulkane des Sonnensystems. Mit einer durchschnittlichen Entfernung von
-228 Millionen Kilometern ist er rund 1,5-mal so weit von der Sonne entfernt wie die Erde.
+eine vielfältige Geologie und die höchsten Vulkane des Sonnensystems. Und das wollen wir erforschen. Mit einer ungefähren Entfernung von
```
Comment (PabloRdrRbl) [inline Mars.txt:6]: German is so difficult.

Comment (ivalaginja) [reply Mars.txt:6]: Ok, how about I translate this first, before we merge?

File: Mars.txt [hunk 2]
```diff
@@ -1,9 +1,12 @@
 Der Mars ist, von der Sonne aus gezählt, der vierte Planet im Sonnensystem und der äußere Nachbar der Erde. Er zählt
 zu den erdähnlichen (terrestrischen) Planeten.
 
-Sein Durchmesser ist mit knapp 6800 Kilometern etwa halb so groß wie der der Erde, sein Volumen beträgt gut ein
+Sein Durchmesser ist mit knapp 6800 Kilometern etwa halb so groß wie der der Erde (Earth), sein Volumen beträgt gut ein
```
Comment (PabloRdrRbl) [inline Mars.txt:4]: Comment 1

File: Mars.txt (hunk 1 above)
Comment (PabloRdrRbl) [inline Mars.txt:6]: Comment 2

**Review by PabloRdrRbl — COMMENTED** [review]
Here my review

File: Mars.txt (hunk 1 above)
Comment (PabloRdrRbl) [reply Mars.txt:6]: It's ok.

**Pull request closed without merging**